pkg/odo/cli/service/create.go

```diff
@@ -195,10 +227,15 @@ func NewCmdServiceCreate(name, fullName string) *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			util.LogErrorAndExit(o.Complete(name, cmd, args), "")
 			util.LogErrorAndExit(o.Validate(), "")
-			util.LogErrorAndExit(o.Run(), "")
+			equivalent := o.outputNonInteractiveEquivalent()
+			if len(equivalent) > 0 {
+				log.Info("Equivalent command:")
+				fmt.Println(ui.StyledOutput(equivalent, "cyan"))
```

Comment:
I separated it because `log.Info` outputs in bold and I wanted to be in regular weight in addition of being in a different color in case the color doesn't work well or is not picked up by users.